MapboxDirections/MBVisualInstruction.swift

```diff
@@ -91,14 +93,14 @@ open class VisualInstruction: NSObject, NSSecureCoding {
         
         self.textComponents = textComponents
         
-        self.degrees = decoder.decodeDouble(forKey: "degrees")
+        self.finalHeading = decoder.decodeDouble(forKey: "degrees")
     }
     
     public func encode(with coder: NSCoder) {
         coder.encode(text, forKey: "text")
         coder.encode(maneuverType, forKey: "maneuverType")
         coder.encode(maneuverDirection, forKey: "maneuverDirection")
-        coder.encode(degrees, forKey: "degrees")
+        coder.encode(finalHeading, forKey: "degrees")
```

Comment:
OK. What does this mean for all the other keys that already match the renamed properties?

Reply:
It would just mean a bit more re-mapping but I'm hoping the contract would be more strongly maintained by a protobuf definition before this becomes relevant.